arb_os/evmCallStack.mini

```diff
@@ -835,6 +860,60 @@ public impure func restoreAuxStackAndCall(
     }
 }
 
+public impure func saveAuxStack() -> Stack {
```

Comment:
It's odd that these register as newly added by the PR when they already exist in the repo

Reply:
Not sure what that is reported as new.